modules/apps/depot/depot-test/src/testIntegration/java/com/liferay/depot/internal/security/permission/contributor/test/DepotRoleContributorTest.java

```diff
@@ -54,6 +57,37 @@ public void setUp() throws Exception {
 		_group = GroupTestUtil.addGroup();
 	}
 
+	@FeatureFlags("LPD-17564")
+	@Test
+	public void testCMSConsumerRoleAssignment() throws Exception {
+		Group group = _groupLocalService.fetchGroup(
```

Comment:
@AliciaGarciaGarcia, the logic is a bit weird when the group doesn't exist..... Can't we create the group with the groupKey at once instead of updating the other _group and later updating it?

We have

```
public static Group addGroup(
			long companyId, long userId, long parentGroupId, String groupKey)
		throws Exception {
```

in GroupTestUtil for example

Reply:
done